docs/user-guide/certificates-configuration-questionnaire.md

```diff
@@ -0,0 +1,93 @@
+# Zowe certificates configuration questionnaire
+
+To properly configure Zowe to use certificates for server-side component installation review the various certificate setup options presented in this article. 
+Understanding these options makes it possible to select the best certificate configuration scenario that fits your Zowe deployment use case. 
+Review the Configure Zowe Certificates diagram and  answer the questions in the questionnaire at the end of this article.
+
+**Tip:**
+Before determining which scenario best suits your use case, it is practical to have a general understanding of the certificate configuration basics and Zowe certificates configuration overview. For more information, see the following articles: 
+- [Certificates concepts](../appendix/zowe-security-glossary#certificate-concepts) in the [Zowe Security Glossary](../appendix/zowe-security-glossary)
+- [Zowe certificates overview](../getting-started/zowe-certificates-overview)
+
+
+The numerated decision blocks (the yellow diamonds) in the following diagram correspond to the questions in the questionnaire.
+Follow this sequence of questions to determine which certificate configuration scenario best suits your certificate use case.
+
+![Certificates configuration decision tree](../images/install/certificates-config-scenarios.png)
+
+Each of the following certificate configuration scenarios are available in the article [Certificate configuration scenarios](./certificate-configuration-scenarios.md).
+
+* [Scenario 1: Use a file-based (PKCS12) keystore with Zowe generated certificates](./certificate-configuration-scenarios/#scenario-1-use-a-file-based-pkcs12-keystore-with-zowe-generated-certificates)
+* [Scenario 2: Use a file-based (PKCS12) keystore and import a certificate generated by another CA](./certificate-configuration-scenarios/#scenario-2-use-a-pkcs12-keystore-and-import-a-certificate-generated-by-another-ca)
+* [Scenario 3: Use a z/OS keyring-based keystore with Zowe generated certificates](./certificate-configuration-scenarios/#scenario-3-use-a-zos-keyring-with-zowe-generated-certificates)
+* [Scenario 4: Use a z/OS keyring-based keystore and connect an existing certificate](./certificate-configuration-scenarios/#scenario-4-use-a-zos-keyring-and-connect-to-an-existing-certificate)
+* [Scenario 5: Use a z/OS keyring-based keystore and import a certificate stored in a data set](./certificate-configuration-scenarios/#scenario-5-use-a-zos-keyring-and-import-a-certificate-stored-in-a-data-set)
+ 
+## Certificate configuration questionnaire 
```

Comment:
It's hard to understand what's the value here. I always get to many scenarios. How does it help me select one, even more so compared to the image, which clearly leads to one scenario as the solution to the specific subset of problems.

Reply:
To be resolved in an improvement task.